openapi/src/parts/workspace.yaml

```diff
@@ -356,6 +356,34 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/ServerError'
+
+  /api/workspaces/v1/{workspaceId}/policies/mergeCheck:
+    parameters:
+    - $ref: '#/components/parameters/WorkspaceId'
+    post:
+      summary: The result of a merge calculation between two workspaces. This does not mutate the original workspaces.
+      description: |
+        The target workspace is identified by the path parameter.
+        The source workspace is provided in the source request. 
+        Policies from the source workspace are merged into the target workspace.
+        The merge operation is called with DRY_RUN, so no mutations take place.
+      operationId: mergeCheck
+      tags: [ Workspace ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MergeCheckRequest'
+      responses:
+        '200':
```

Comment:
Since there is no response body, this would be a 204 - NO_CONTENT

I think you need to specify 409 - CONFLICT, since that is what you are throwing when there is a conflict.

I'll leave it to @cbookg to say whether getting an error message is sufficient, or if you would like a response body with more explanation for the conflict.

The TPS response for merge is a 200 even when there is a conflict so that we can return the conflict information on the response.

Reply:
After talking with Chris, we decided to always return a successful response with a response body (even when there are conflicts)

So, the endpoint now returns:
1. a list of conflicts (between the workspace policies themselves)
2. a list of existing resources with conflicts (if the other workspace was to be merged into the target)

Then, the UI can infer what to do next.